src/os_execd/win_execd.c

```diff
@@ -109,6 +134,9 @@ void WinExecdRun(char *exec_msg)
 
     timeout_data *timeout_entry;
 
+    /* Delete pending AR at succesfull exit */
+    atexit(WinExecd_Shutdown);
```

Comment:
Calling `atexit()` at this point will register the same times. I suggest you move it to WinExecd_Start().